module-3/web-app/src/components/Pages/Documents/Documents.module.scss

```diff
@@ -0,0 +1,78 @@
+.container {
+    display: flex;
+    flex-wrap: wrap;
+    align-content: flex-start;
+    padding: .5rem 2.3rem;
+}
+
+.fileIcon {
```

Comment:
`fileIcon` -> `file-icon`

Reply:
A few comments upper you asked to rename it as file-icon-wrapper.